modules/perception/onboard/component/lane_detection_component.cc

```diff
@@ -610,10 +602,9 @@ int LaneDetectionComponent::InternalProc(
   Eigen::Affine3d camera2world_trans;
   if (!camera2world_trans_wrapper_map_[camera_name]->GetSensor2worldTrans(
           msg_timestamp, &camera2world_trans)) {
-    std::string err_str = "failed to get camera to world pose, ts: " +
-                          std::to_string(msg_timestamp) +
-                          " camera_name: " + camera_name;
-    AERROR << err_str;
+    AERROR << "Failed to get camera to world pose, ts: "
+           << std::to_string(msg_timestamp) << ", camera_name: " << camera_name;
```

Comment:
No need to convert to string in a streaming. Please also fix all other such cases.